packages/desktop/src/renderer/components/widgets/channels/ChannelInput/ChannelInput.tsx

```diff
@@ -268,6 +277,15 @@ export const ChannelInputComponent: React.FC<ChannelInputProps> = ({
     messageRef.current = message
   }, [message])
 
+  React.useEffect(() => {
+    if(Object.values(messages).length >= 1 && inputState === INPUT_STATE.AVAILABLE){
```

Comment:
Instead of adding logic in the component, I think you should only pass INPUT_STATE down to it